### ISSUE TITLE
The Back button is not available after returning from home screen

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-tv#2218]
## Steps to reproduce
1. Launch Firefox.
2. Open a website by entering a URL into the search bar.
3. Open browser overlay by pressing the menu button.
4. Press the Home button.
5. Return to Firefox and the overlay browser is displayed.

### Expected behavior
The `Back` button is available and after tapping it, it returns FF to the previous website visited.
### Actual behavior
The `Back` button is not available.
### Device information
* Fire TV device: **`Fire Stick 2nd gen`**
* Latest affected Firefox version: **`Firefox TV v3.9 LAT2`**
* Earliest affected Firefox version: **`Firefox TV v3.8`**
### Notes
For more details please see the [logcat.txt](https://github.com/mozilla-mobile/firefox-tv/files/3162319/logcat.txt) and [video](https://drive.google.com/file/d/1pgfZackoy11czcxV_VExCjG9YcJ7ms72/view?usp=sharing)

**Comment by npark-mozilla** [comment]
@abodea curious, if you only visited one site, I don't think back button would  be active?

**Comment by abodea** [comment]
@npark-mozilla Yes in the video example I visited only one site, If I visit more sites I will have the back button but it will redirect me to the previously visited site.
I reported this in the idea that no matter how many sites I have opened, once I'm in menu overlay and 1 website was opened, pressing the back button would redirect me to that website.

**Comment by kglazko** [comment]
At the end of the video I see you pressing refresh and the URL is rotten tomatoes, nothing happens then and it is stuck on home screen and doesn’t go to Rotten tomatoes?

**Comment by abodea** [comment]
@kglazko no, I just selected the **`Refresh`** button to make a clear view for the header bar that the **`Back`** button is not available. 
The **`Refresh`** button is only selected.
